src/parser/monk/windwalker/modules/core/GlobalCooldown.js

```diff
@@ -0,0 +1,22 @@
+import CoreGlobalCooldown from 'parser/shared/modules/GlobalCooldown';
+import SPELLS from 'common/SPELLS';
+
+/**
+ * The Global Cooldown of Fists of Fury is also being triggered by fabricated channeling events
+ */
+class GlobalCooldown extends CoreGlobalCooldown {
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    const isOnGCD = this.isOnGlobalCooldown(spellId);
+    if (spellId === SPELLS.FISTS_OF_FURY_CAST.id) {
```

Comment:
I should've clarified a little more - I meant to do the spellId check before even calling `isOnGlobalCooldown`, so this way:
```javascript
on_byPlayer_cast(event) {
    const spellId = event.ability.guid;
    if (spellId === SPELLS.FISTS_OF_FURY_CAST.id) {
      return;
    }
    const isOnGCD = this.isOnGlobalCooldown(spellId);
    if (!isOnGCD) {
      // This ensures we don't crash when boss abilities are registered as casts which could even happen while channeling. For example on Trilliax: http://i.imgur.com/7QAFy1q.png
      return;
    }
    super.on_byPlayer_cast(event);
  }
```
This way, if it's a FoF cast, it doesn't even have to execute the function.

Reply:
Yeah, i understood the reasoning for the change, just oversaw that it was already executed when assigned. It should be all good now.